plugins/obs-filters/data/color_correction_filter.effect

```diff
@@ -0,0 +1,97 @@
+/******************************************************************************
+ * Written for OBS by c3r1c3
+ * Based on code from
+ * http://gamedev.stackexchange.com/questions/28782/hue-saturation-brightness-contrast-effect-in-hlsl
+ ******************************************************************************/
+
+uniform float4x4 ViewProj;
+uniform texture2d image;
+
+uniform float3 gamma;
+uniform float3 color_curve;
+uniform float contrast;
+uniform float brightness;
+uniform float saturation;
+uniform float opacity;
+
+/* Pre-Compute variables. */
+uniform float3 rotational_matrix_row1;
+uniform float3 rotational_matrix_row2;
+uniform float3 rotational_matrix_row3;
+
+
+sampler_state textureSampler {
+	Filter   = Linear;
+	AddressU = Clamp;
+	AddressV = Clamp;
+};
+
+struct VertData
+{
```

Comment:
Style:  Opening curly brace ( { ) should go on the same line as the struct definition.  See also other similar lines below.